charts/fybrik/files/taxonomy/base_taxonomy.json

```diff
@@ -0,0 +1,190 @@
+{
+  "title": "taxonomy.json",
+  "definitions": {
+    "Action": {
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/ActionName"
+        }
+      },
+      "additionalProperties": true
+    },
+    "ActionName": {
+      "type": "string"
+    },
+    "AppInfo": {
+      "type": "object",
+      "additionalProperties": true
+    },
+    "AssetID": {
+      "description": "Asset ID of the asset to be queried in the catalog",
+      "type": "string"
+    },
+    "AttributeType": {
+      "type": "string",
+      "enum": [
+        "numeric",
+        "string",
+        "bool"
+      ]
+    },
+    "Capability": {
+      "type": "string"
+    },
+    "Connection": {
+      "description": "Name of the connection to the data source",
```

Comment:
You could mention that the details of the connection are meant to be defined in additional layers

Reply:
OK